content/home/manifesto.md

```diff
@@ -2,7 +2,7 @@
 title: "Project Manifesto"
 summary: "What is this project about?"
 date: 2022-02-11
-weight: 2
+weight: 1
 ---
 
 Our goal is to create an open standard for feature flag management to support a robust feature flag ecosystem using cloud native technologies.
```

Comment:
Can we make this "The goal of the open feature project ..."  not that people confuse overall project goals with individual goals

Reply:
```suggestion
The goal of the OpenFeature project is to create an open standard for feature flag management to support a robust feature flag ecosystem using cloud native technologies.
```